force_wfmanager/central_pane/setup_pane.py

```diff
@@ -0,0 +1,38 @@
+from pyface.tasks.api import TraitsTaskPane
+
+from traits.api import Instance, Dict
+
+from traitsui.api import View, VGroup, UItem
+from traitsui.editors import ShellEditor
+
+from .analysis_model import AnalysisModel
+
+
+class SetupPane(TraitsTaskPane):
+    id = 'force_wfmanager.setup_pane'
+    name = 'Setup Pane'
+
+    #: The model for the analysis part
+    analysis_model = Instance(AnalysisModel)
+
+    # Namespace for the console
+    console_ns = Dict()
+
+    view = View(VGroup(
+            UItem("console_ns", label="Console", editor=ShellEditor()),
+    ),
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(SetupPane, self).__init__(*args, **kwargs)
+
+    def _console_ns_default(self):
+        namespace = {
+            "task": self.task
+        }
+        try:
```

Comment:
this smells a bit. 

First, why do you need the `app` in the console? Is it for debugging? Second, you should probably rethink this and just contribute items to the console. There is a plugin supporting that in envisage IIRC.

This should not hold up this PR but ...

Reply:
I think it was already passed as a keyword argument anyway, so the __init__ was definitely unneccesary! It would be good to know how the traits initialisation works though. I was working on the assumption that traits magic would happen during `super(TraitSubclass, self).__init__()` but maybe this isn't completely true? Can you think of anything to watch out for if I'm doing this sort of thing?
For the console, it's just there for debugging and will eventually only be displayed in the applications 'debug' mode. I'm not too familiar with the details of how the console is set up though.